Modules/_sqlite/statement.c

```diff
@@ -28,6 +28,10 @@
 #include "prepare_protocol.h"
 #include "util.h"
 
+#if SQLITE_VERSION_NUMBER >= 3007011
```

Comment:
I understand why did you prefer this way, but it would be much better if we could defer feature checking to `configure`.

Reply:
I am following the pattern elsewhere in this module, for example in connection.c there have been recent changes to support the sqlite3 backup APIs, which use this same pattern.